modules/signals/tsconfig.spec.json

```
@@ -2,10 +2,10 @@
   "extends": "../../tsconfig.json",
   "compilerOptions": {
     "outDir": "../../dist/out-tsc",
-    "module": "commonjs",
-    "types": ["jest", "node"],
+    "module": "es2022",
+    "types": ["jest", "node", "vitest", "vitest/globals"],
```

Nit: Can we also remove `jest` here?
 
```suggestion
    "types": ["node", "vitest", "vitest/globals"],
```